cnjokes/src/components/Layout.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Fragment } from "react";
+import { AppNavbar } from "./AppNavbar/AppNavbar";
+
+class Layout extends React.Component {
```

Comment:
Please use destructuring of `Component` in import section;
`import React, { Component } from "react";`
`class Layout extends Component {`